lib/ACL/RuleManager.php

```diff
@@ -293,6 +295,26 @@ public function saveRule(Rule $rule): void {
 				->andWhere($query->expr()->eq('mapping_type', $query->createNamedParameter($rule->getUserMapping()->getType())))
 				->andWhere($query->expr()->eq('mapping_id', $query->createNamedParameter($rule->getUserMapping()->getId())));
 			$query->executeStatement();
+
+			if ($rule->getUserMapping()->getType() === 'user') {
+				$logMessage = 'The ACL rule was updated to permission "%s" ans mask "%s" for file/folder with id: "%s" for user "%s"';
```

Comment:
s/ans/and/ all messages

I'd also remove the colon to make it flow better

Reply:
done